mw-webapp/src/logic/wayPage/reportsTable/WayColumns.module.scss

```diff
@@ -19,16 +19,15 @@ $listLeftMargin: 20px;
 }
 
 .flatButton {
+  width: 100px;
```

Comment:
Why do you need to specify width here if it is the same default width for Button component for now?